lib/Sabre/Album/AlbumPhoto.php

```diff
@@ -26,22 +26,28 @@
 use OCA\Photos\Album\AlbumFile;
 use OCA\Photos\Album\AlbumInfo;
 use OCA\Photos\Album\AlbumMapper;
+use OCA\Photos\Sabre\CollectionPhoto;
 use OCP\Files\IRootFolder;
 use OCP\Files\Node;
 use OCP\Files\File;
+use OCP\Files\Folder;
 use OCP\Files\NotFoundException;
-use Sabre\DAV\Exception\Forbidden;
 use Sabre\DAV\IFile;
 
-class AlbumPhoto implements IFile {
+class AlbumPhoto extends CollectionPhoto implements IFile {
```

Comment:
Review hint: some bits extracted to `CollectionPhoto` so it can be shared with `LocationPhoto`